deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -507,9 +510,9 @@ spec:
         - name: KUBERNETES_META_REDUCE
           value: "false"
         - name: MULTILINE_START_REGEXP
-          value: "/^\\w{3} \\d{1,2}, \\d{4}/"
+          value: 
```

Comment:
Since these aren't being used anymore, can you also remove these two from `deploy/helm/sumologic/templates/deployment.yaml` so that they're no longer rendered here?